src/gt4py/next/ffront/func_to_past.py

```diff
@@ -162,6 +162,19 @@ def visit_BinOp(self, node: ast.BinOp, **kwargs: Any) -> past.BinOp:
     def visit_Name(self, node: ast.Name) -> past.Name:
         return past.Name(id=node.id, location=self.get_location(node))
 
+    def visit_Attribute(self, node: ast.Attribute) -> past.AttributeExpr:
```

Comment:
For safety it could make sense to check that `ctx` is `Load` and give a proper error message in the other cases, see https://docs.python.org/3/library/ast.html#ast.Attribute for context.

Reply:
Makes sense. Should this be down here on in `type_deduction` ?

Reply:
I would do it here, then we don't need to carry around the `ctx` information.